### PR TITLE
Add uploaded dropdown in media library for image uploads

### PR DESCRIPTION
**Pull request by rawcreative**
Adds the 'Uploaded to this Post' dropdown option in the media modal when selecting images when using the Upload or Multi-Upload option types.

### CODE REVIEW

File: framework/includes/option-types/multi-upload/static/js/images-only.js [hunk 1]
```diff
@@ -31,7 +31,14 @@
 					library: {
 						type: 'image'
 					},
-					multiple: true
+					multiple: true,
+					states: new wp.media.controller.Library({
+                        library:   wp.media.query( { type: 'image' } ),
+                        multiple:  true,
+                        title:     'Select Images',
```
Comment (moldcraft) [inline framework/includes/option-types/multi-upload/static/js/images-only.js]: This should be translatable

File: framework/includes/option-types/upload/static/js/images-only.js [hunk 2]
```diff
@@ -23,7 +23,14 @@
 				frame = wp.media({
 					library: {
 						type: 'image'
-					}
+					},
+					states: new wp.media.controller.Library({
+                        library:   wp.media.query( { type: 'image' } ),
+                        multiple:  false,
+                        title:     'Select Image',
```
Comment (moldcraft) [inline framework/includes/option-types/upload/static/js/images-only.js]: This should be translatable

**Comment by rawcreative** [comment]
Updated PR with translation capability

**Comment by moldcraft** [comment]
Minor: Your changes are with space indentation, must be with tab indentation.

**Pull request merged**